src/components/PlayerSearch.vue

```diff
@@ -105,6 +92,13 @@ export default {
     },
   },
   methods: {
+    getArray(num) {
+      return this.players.result.slice((num - 1) * 10,
+        Math.min((num - 1) * 10 + 10, this.players.result.length));
+    },
+    goTo(uri) {
```

Comment:
nit, this is not a full uri, but a path.
Do we really need this method? Does something like `to=` work? Otherwise, we can directly call the router push?